app/views/tracks/index.html.haml

```diff
@@ -6,38 +6,84 @@
+      %p.text-right
+        = link_to '#status-help', class: 'btn btn-default',  "data-toggle"=>"collapse" do
+          Help?
+      .collapse#status-help
+        %p
+          %strong
+            What happens next with my track request?
+        %p
+          If you submit a track request, the conference organizers will review it and either accept or reject it.
+          %br
+          If your track request is accepted, the conference organizers expect you to confirm that you will be able to hold it.
+          Then you will gain the Track organizer role.
+          %br
+          If your track request is rejected, you can either live with that or adapt it and resubmit it for review again.
+          %br
+          If something changes and you can't organize the track any more, you should withdraw it.
+
+        %p
+          %strong
+            Why do I need to add more information?
+        %p
+          The more information you add to your proposal, the more likely it is that the conference organizers accept your track request.
+
+        %p
+          %strong
+            Why do I need to register to the conference?
+        %p
+          Knowing the number of visitors for the conference helps the organizers plan better.
+
   - if @tracks.any?
     .row
       .col-md-12
-        %table.table.table-hover#tracks
-          %thead
-            %th Name
-            %th Short name
-            %th Description
-            %th Color
-            %th State
-            %th Actions
-          %tbody
-            - @tracks.each do |track|
-              %tr
-                %td
-                  = link_to(conference_program_track_path(@conference.short_title, track)) do
-                    = track.name
-                %td
-                  = track.short_name
-                %td
-                  %p
-                    = truncate(track.description)
-                %td
-                  %span.label{style: "background-color: #{track.color}; color: #{ contrast_color(track.color) }"}
-                    = track.color
-                %td
-                  = track.state
-                %td
+        %table.table.table-striped#tracks
+          - @tracks.each do |track|
+            %tr
+              %td{style: "padding:15px 0px 0px 8px;"}
+                - if %w(new to_accept to_reject).include? track.state
+                  %span{ title: 'In review', class: 'fa fa-eye' }
+                - elsif track.state == 'accepted'
+                  %span{ title: 'Accepted', class: 'fa fa-check text-muted' }
+                - elsif track.state == 'confirmed'
+                  %spam{ title: 'Confirmed', class: 'fa fa-check text-success' }
+                - elsif %w(rejected withdrawn canceled).include? track.state
+                  %span{ title: track.state.humanize, class: 'fa fa-ban'}
+              %td
+                = link_to(conference_program_track_path(@conference.short_title, track)) do
+                  = track.name
```

Comment:
What about adding a background with the track's color, instead of having a separate column for it?